templates/resources/lambda_function.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Basic example
+To create a AWS lambda function with basic details
+{{ tffile (printf "examples/resources/%s/lambda_basic.tf" .Name)}}
+
+### Lambda Layer example
+To create a AWS lambda function using lambda layers
+{{ tffile (printf "examples/resources/%s/lambda_layer.tf" .Name)}}
+
+### Ephemeral storage example
+To create a AWS lambda function using Ephemeral storage
+{{ tffile (printf "examples/resources/%s/lambda_ephemral_storage.tf" .Name)}}
```

Comment:
typo on the file name, I suggest to rename both the file and the path here
`lambda_ephemral_storage.tf`

Reply:
updated filename and filepath from `lambda_ephemral_storage.tf` to `lambda_ephemeral_storage.tf`.